typeDefs.js

```diff
@@ -127,6 +127,19 @@ const typeDefs = gql`
     additionalPrice: Int
   }
 
+  type UserForComment {
+    email: String!
+    name: String
+    images: ImageSet
+    isAdmin: Boolean
+  }
+
+  type UserRate {
+    user: User!
+    rate: Int
```

Comment:
It should not be nullable as lastUpdatedDate too

Reply:
This field is an afterthought for potential use in BI, it's not vital for the work of application. Should default value not be recorded per mongoose model or someone want to modify the collection structure without bringing the website down, an extra non-nullable field will be a pain. Is there a benefit to making it such that I can't see?

Reply:
rate updated, lastUpdatedDate removed